js/main.js

```diff
@@ -1,2 +1,47 @@
 /* global data */
 /* exported data */
+
+var imgInput = document.getElementById('imageInput');
+var imgOutput = document.getElementById('imageOutput');
+var form = document.querySelector('form');
+
+var data = {
+  view: 'entry-form',
+  entries: [],
+  editing: null,
+  nextEntryId: 1
+};
+
+var storage = localStorage.getItem('code-journal');
+if (storage !== null) {
+  data = JSON.parse(storage);
+}
+
+imgInput.addEventListener('blur', function (event) {
+  var imgURL = imgInput.value;
+  imgOutput.src = imgURL;
+});
+
+form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var title = event.currentTarget[0].value;
+  var photoUrl = event.currentTarget[1].value;
+  var notes = event.currentTarget[2].value;
```

Comment:
You should use the `elements` property on the form, so you can get these by name. It will make your code easier to read.